owncloudData/src/main/java/com/owncloud/android/data/files/repository/OCFileRepository.kt

```diff
@@ -423,20 +423,22 @@ class OCFileRepository(
 
                         })
                 } else {
-                    // File exists in the database, we need to check several stuff.
-                    folderContentUpdated.add(
-                        remoteChild.apply {
-                            copyLocalPropertiesFrom(localChildToSync)
-                            // DO NOT update etag till contents are synced.
-                            etag = localChildToSync.etag
-                            needsToUpdateThumbnail =
-                                (!remoteChild.isFolder && remoteChild.modificationTimestamp != localChildToSync.modificationTimestamp) || localChildToSync.needsToUpdateThumbnail
-                            // Probably not needed, if the child was already in the database, the av offline status should be also there
-                            if (remoteFolder.isAvailableOffline) {
-                                availableOfflineStatus = AVAILABLE_OFFLINE_PARENT
-                            }
-                            // FIXME: What about renames? Need to fix storage path
-                        })
+                    if (localChildToSync.etag != remoteChild.etag) {
```

Comment:
This is an `else if`, KtLint will probably complain about this

Reply:
Changed, although Ktlint wasn´t complaining